wallet/src/de/schildbach/wallet/ui/BuyAndSellLiquidUpholdActivity.kt

```diff
@@ -126,7 +126,7 @@ class BuyAndSellLiquidUpholdActivity : LockScreenActivity() {
         updateBalances()
 
         liquid_container.setOnClickListener {
-            analytics.logEvent(if (UpholdClient.getInstance().isAuthenticated) {
+            analytics.logEvent(if (LiquidClient.getInstance()?.isAuthenticated == true) {
```

Comment:
This was a bug in logging the Liquid event, no idea how I screwed this up.

Reply:
This will probably conflict with `feature-coinbase-integration`.  We will have to make sure the change is correctly reflected when `master` is merged into `feature-coinbase-integration`.